go.mod

```diff
@@ -29,7 +29,7 @@ require (
 	github.com/grpc-ecosystem/grpc-opentracing v0.0.0-20180507213350-8e809c8a8645
 	github.com/hashicorp/go-hclog v1.3.1
 	github.com/hashicorp/go-plugin v1.4.5
-	github.com/kr/pretty v0.3.0
+	github.com/kr/pretty v0.3.1
```

Comment:
Please exclude go.mod changes from this PR, it's better to commit dependency upgrades separately.

Reply:
I updated my branch, seems that was an update issue as those changes were not from my PR. You may please check now.